tuf/api/pylintrc

```diff
@@ -0,0 +1,9 @@
+[BASIC]
+good-names=e, fn, fp
```

Comment:
Minor nit: Right now we don't use those names. What do you think about removing the line and only add when needed, or when fleshing out the config in order to align with the style guide TBD (https://github.com/secure-systems-lab/code-style-guidelines/issues/20)?

Reply:
Completely agree, I'll remove this line.